templates/site/torrents/view.jet.html

```diff
@@ -64,24 +64,24 @@ <h1 style="text-align: center;" class="torrent-hr">{{Torrent.Name}}</h1>
       </td>
     </tr>
     {{end}}
-	{{ if len(Torrent.Tags) != 0 || User.ID > 0 }}
+	 {{ if Torrent.Tags.HasAccepted() || User.ID > 0 }}
     <tr class="torrent-info-row">
       <td class="torrent-info-td torrent-info-label">{{ T("torrent_tags")}}:</td>
       <td class="tr-flag torrent-view-td torrent-info-data">
-        {{ range Torrent.Tags}}
+        {{ range Torrent.Tags }}
+		{{if !.Accepted || User.ID > 0 }}
         <span class="tag{{ if len(Torrent.Tags) > 5 }} big{{ end }}{{ if !.Accepted }} pending{{else}} accepted{{end}}" title="Tag: {{ .Type }} ({{ if !.Accepted }}{{ .Total }}{{else}}{{ T("accepted") }}{{end}})">
           <span class="tag-text{{ if User.ID > 0 }} votable{{end}}">
```

Comment:
Yes i do, if i understood it right there may be both Accepted and Pending tags for the same torrent